xml/System/IntPtr.xml

```diff
@@ -824,8 +824,17 @@
       </ReturnValue>
       <Docs>
         <summary>Represents the largest possible value of <see cref="T:System.IntPtr" />.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <value>
+          <paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</value>
```

Comment:
```suggestion
        <value>To be added.</value>
```

Reply:
@tannergooding  This needs to be addressed in triple slash comments so the porting tool doesn't keep trying to port it on every preview.